pkg/ipamd/ipamd.go

```diff
@@ -692,9 +719,19 @@ func (c *IPAMContext) increaseDatastorePool(ctx context.Context) {
 		return
 	}
 	// Try to add more Cidrs to existing ENIs first.
+	if c.inInsufficientCidrCoolingPeriod() {
+		log.Debugf("Recently we had InsufficientCidr error hence will wait for %v mins before retrying", insufficientCidrErrorCooldown)
```

Comment:
let's remove the mins, it will render as 2m0s already by the "%v"

Reply:
sure